nuxeo-layout.html

```diff
@@ -84,8 +84,34 @@
       validate: function() {
         if (this.element && ('function' === typeof this.element.validate)) {
           return this.element.validate();
+        } else {
+          // workaroud for https://github.com/PolymerElements/iron-form/issues/218, adapted from iron-form.html
+          var valid = true;
+          if (this.element) {
+            var elements = this._getValidatableElements(this.element.root);
+            for (var el, i = 0; el = elements[i], i < elements.length; i++) {
+              valid = !!el.validate() && valid;
+            }
+          }
+          return valid;
+        }
+      },
+
+      _getValidatableElements: function(parent) {
+        var nodes = Polymer.dom(parent).querySelectorAll('*');
+        var submittable = [];
+        for (var i = 0; i < nodes.length; i++) {
+          var node = nodes[i];
+          if (!node.disabled) {
+            if (node.validate) {
+              submittable.push(node);
```

Comment:
Aren't we leaving out all the native validatable inputs? or we don't care because we won't trigger native validation anyway?

Reply:
We can check for validity of individual native elements, but it won't actually trigger native HTML5 form validation.